### PR TITLE
Upgrade to GeoIP2 City

### PR DESCRIPTION
**Pull request by firstred**
| Questions | Answers |
| --- | --- |
| Branch? | `develop` |
| Description? | Upgrade from legacy to MaxMind's GeoIP2 database. |
| Type? | improvement |
| Category? | CO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? |  |
| How to test? |  |

### CODE REVIEW

**Comment by julienbourdeau** [comment]
Great!

**Comment by xBorderie** [comment]
Super duper!

File: config/defines.inc.php [hunk 1]
```diff
@@ -94,10 +94,10 @@
 
 define('_PS_TOOL_DIR_', _PS_CORE_DIR_.'/tools/');
 if (!defined('_PS_GEOIP_DIR_')) {
-    define('_PS_GEOIP_DIR_', _PS_TOOL_DIR_.'geoip/');
+    define('_PS_GEOIP_DIR_', _PS_CORE_DIR_.'/localization/');
```
Comment (julienbourdeau) [inline config/defines.inc.php]: Can you move it to `app/Resources/geoip`?
This folder is only used for former XMLs, we're trying to move everything to `app/Resources` and these files will be migrated to CLDR data using standards locale (at some point :D)

File: controllers/admin/AdminGeolocationController.php [hunk 2]
```diff
@@ -140,8 +140,8 @@ public function initContent()
         $this->display = 'options';
         if (!$this->isGeoLiteCityAvailable()) {
             $this->displayWarning($this->l('In order to use Geolocation, please download').'
-				<a href="http://geolite.maxmind.com/download/geoip/database/GeoLiteCity.dat.gz">'.$this->l('this file').'</a> '.
-                $this->l('and extract it (using Winrar or Gzip) into the /tools/geoip/ directory.'));
+				<a href="http://geolite.maxmind.com/download/geoip/database/GeoLite2-City.mmdb.gz" target="_blank">'.$this->l('this file').'</a> '.
+                $this->l('and extract it (using Winrar or Gzip) into the /localization/ directory.'));
```
Comment (julienbourdeau) [inline controllers/admin/AdminGeolocationController.php]: Same here of course

File: config/defines.inc.php (hunk 1 above)
Comment (firstred) [inline config/defines.inc.php]: That's what I needed to know. Thank you :smile:

**Comment by firstred** [comment]
Just moved it and it still works :scream:

**Comment by firstred** [comment]
And a dummy `index.php` because git doesn't know folders, obviously.

**Comment by mickaelandrieu** [comment]
Thank you @firstred

**Pull request merged**